plugins/modules/ovirt_disk.py

```diff
@@ -589,7 +594,16 @@ def build_entity(self):
             ) if logical_unit else None,
         )
         if hasattr(disk, 'initial_size') and self._module.params['upload_image_path']:
-            disk.initial_size = size
+            out = subprocess.check_output([
+                'qemu-img',
+                'measure',
+                '-f', 'qcow2' if self._module.params.get('format') == 'cow' else 'raw',
+                '-O', 'qcow2' if self._module.params.get('format') == 'cow' else 'raw',
+                '--output', 'json',
+                self._module.params['upload_image_path']
+            ])
+            measure = json.loads(out)
+            disk.initial_size = measure["required"]
```

Comment:
measure["required"] is not a boolean value?
disk.initial_size shouldn't be a number?

Reply:
Did you try to run qemu-img measure? It will answer your question.

Reply:
OK. No issues :)
The JSON output is an object of QAPI type BlockMeasureInfo
Fields
fully_allocated: isize
required: isize